ENGAGEHF/HeartHealth/SelectionTypes/GraphSelection.swift

```diff
@@ -0,0 +1,88 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import FirebaseFirestore
+import Foundation
+import HealthKit
+
+
+/// The type of Vitals to be displayed as the main content of the Heart Health view
+/// Chosen by GraphPicker in HeartHealth
+enum GraphSelection: CaseIterable, Identifiable, CustomStringConvertible, Equatable {
+    case symptoms
+    case weight
+    case heartRate
+    case bloodPressure
+    
+    
+    var id: Self {
+        self
+    }
+    
+    var description: String {
+        switch self {
+        case .symptoms: "Symptoms"
```

Comment:
I made an issue documenting this! Shouldn't be too bad, would essentially be using the DecodeLocalizedString extension instead of default String decoding everywhere we're decoding elements from Firestore.